models/model_test.js

```diff
@@ -6,4 +6,19 @@ const fetchTopics = () => {
     });
 }
 
-module.exports = fetchTopics;
\ No newline at end of file
+const fetchArticleById = (article_id) => {
+
+    if(isNaN(article_id) || article_id <= 0){
+        return Promise.reject({ status: 400, message: 'bad request'})
```

Comment:
Just an fyi, postgres should handle our article not being a number for us. For this to work should you want to refactor later, you might need to check for another code in your error handling middleware though!